client/a8c-for-agencies/components/items-dashboard/items-dataviews/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { Spinner } from '@automattic/components';
+import { DataViews } from '@wordpress/dataviews';
+import { useTranslate } from 'i18n-calypso';
+import { ReactNode } from 'react';
+import ReactDOM from 'react-dom';
+import { ItemsDataViewsType, DataViewsColumn } from './interfaces';
+// todo: Extract from style.scss not common styles (colors and specific to Jetpack Cloud components)
+import './style.scss';
+
+const getIdByPath = ( item: object, path: string ) => {
+	const fields = path.split( '.' );
+	let result: any = item;
```

Comment:
```suggestion
	let result: object = item;
```
Would this work better? With 'any' my editor is suggesting that 'Unexpected any. Specify a different type', and item is defined as an object.

Reply:
It doesn't work for me, the Typescript linter complains about. I have this alternative that works well with Typescript:

`let result: Record< string, any > = item;`